# "an execution error has been registered"

def log_error(file):
    content = open(file,"a")
    return content.write("an execution error has been registered")
    
    